Newsletters: add #33 (2019-02-12)

harding opened the pull request:
Note: tests expected to fail as this links to the RBF post but it isn't included on this branch.  This can be resolved by either rebasing this on top of that, or by merging that PR first and then re-running the tests.  The link check is our final test and it only reports this error on my local build, so the other tests are passing.

moneyball commented:
utACK. Looks great.

practicalswift commented on _posts/en/newsletters/2019-02-12-newsletter.md:
```diff
@@ -0,0 +1,164 @@
+  contain---proving that the transaction creator had access to all of
+  the information necessary to create a spend (e.g. the private keys).
+  The tool is written in the Rust programming language and uses the
+  increasingly-popular BIP174 Partially-Signed Bitcoin Transaction
```
Nit: Could drop hyphen and write "increasingly popular" instead of "increasingly-popular".

practicalswift commented on _posts/en/newsletters/2019-02-12-newsletter.md:
```diff
@@ -0,0 +1,164 @@
+  The tool is written in the Rust programming language and uses the
+  increasingly-popular BIP174 Partially-Signed Bitcoin Transaction
+  (PSBT) format for interoperability with Bitcoin Core and other Bitcoin
+  tools.  Future plans for the tool include privacy enhancements.
```
Nit: Plans imply future. Could write "Plans for .." instead of "Future plans for ...".

practicalswift commented on _posts/en/newsletters/2019-02-12-newsletter.md:
Nit: Eliminate imply complete. Could write "eliminate" instead of "completely eliminate".

jnewbery commented:
I've rebased this on #102

harding commented on _posts/en/newsletters/2019-02-12-newsletter.md:
Although I fully agree that it's really useful original research, I feel like it's good manners for us to put our own announcements after the announcements of things other people have done.  However, I don't think it really matters, so I've implemented your suggestion as a completely separate final commit, and I leave it to you to either keep or drop that commit at merge time (squashing as you see fit).

jnewbery replied:
I think that's a fair argument. I'll drop the last commit and leave this after the other news items.

bitschmidty replied:
Agreed. 

Newsletter looks great again team!

harding commented:
Two commits added addressing feedback from @practicalswift and @jnewbery (thanks!).  The final commit is designed to be dropped without any loss of content depending on the resolution of [this comment](https://github.com/bitcoinops/bitcoinops.github.io/pull/108#discussion_r255920475).

@jnewbery feel better!

jnewbery commented:
Dropped final commit and squashed fixup commit.
